conda/core/subdir_data.py

```diff
@@ -24,6 +21,9 @@
 except ImportError:  # pragma: no cover
     from .._vendor.boltons.setutils import IndexedSet
 
+from conda.gateways.repodata import (
+    get_cache_control_max_age,  # NOQA backwards compatibility alias
+)
```

Comment:
making pre-commit happy

```suggestion
from conda.gateways.repodata import (  # noqa: F401
    get_cache_control_max_age,  # backwards compatibility alias
)
```